plasma/plasma-spec.ini

```diff
@@ -72,6 +72,20 @@ requires:
     andBool 0 <=Int TOKEN_ID     andBool TOKEN_ID <Int (2 ^Int 160)
     andBool 0 <=Int EXITS_QUEUE_ID    andBool EXITS_QUEUE_ID <Int (2 ^Int 160)
 
+[submitBlock]
+k: #execute => #halt
+statusCode: _ => EVMC_SUCCESS
+output: _ => _
+owner: OPERATOR
```

Comment:
```
owner: OPERATOR
msg_sender: OPERATOR
```

This will conflicts because these two cell defined by the same symbol.

Reply:
ah, I see